tests/test_workspace.py

```diff
@@ -101,12 +101,6 @@ def test_get_workspace_model_default(workspace_factory):
     assert m
 
 
-def test_get_workspace_model_default(workspace_factory):
```

Comment:
why was this removed? we actually need to test that we can access the default model.

Reply:
@kratsg Look at what is above it. It is the exact same test as the one above it. If you meant for them to be different tests then I think you might have accidentally copied and pasted the same one twice.

Reply:
oh weird. I wonder how _that_ happened...